Documentation/README.md

```diff
@@ -0,0 +1,58 @@
+# Documentation
+
+## Usage
```

Comment:
I don’t really think we need this heading. We can just remove it and make all the following headings `##` instead of `###`.